molgenis/molgenis-service-armadillo
# feat: form validation link file

how to test:
- go to [UI](https://preview-armadillo-pr-803.dev.molgenis.org/)
- create link file 
- fill in nothing
- click on save
- see form validation

todo:
